examples/multigpu/README.md

```diff
@@ -0,0 +1,23 @@
+# Multiple GPU Training
+
+## Requirements
+
+```bash
+pip install torchmetrics==0.11.4
+```
+
+## How to run
+
+### Node classification
+
+Run with following (available dataset: "ogbn-products", "ogbn-arxiv")
+
+```bash
+python3 node_classification_sage.py --dataset_name ogbn-products
+```
+
+#### __Results__ with default arguments
+```
+* Test Accuracy of "ogbn-products": ~0.7632
+* Test Accuracy of "ogbn-arxiv": ~0.5397
```

Comment:
The result for `ogbn-arxiv` is quite low. See if adding reverse edges helps.

Reply:
No. Could you try https://docs.dgl.ai/generated/dgl.to_bidirected.html#dgl.to_bidirected

Reply:
`to_bidirectted` + self_loop: `Test accuracy 0.6994` How about this number?

Reply:
This appears much more reasonable. Thanks.